src/index.ts

```diff
@@ -96,15 +103,21 @@ export const initUntarJS = async (): Promise<IUnpackJSAPI> => {
         dataSize
       );
 
-      files[filename] = fileData;
-    }
+      const fileDataCopy = new Uint8Array(Array.from(fileData));
 
+      files[filename] = fileDataCopy;
+    }
+    let newFiles = { ...files };
```

Comment:
Since we are creating the `files` dictionary, this copy of `files` here should have no impact but reduce performances?

Reply:
Fixed